File: lt1051.java
// https://leetcode-cn.com/problems/height-checker/
// height checker
class Solution {
    public int heightChecker(int[] heights) {
        
        int len = heights.length;
        int[] temp = new int[len];
        System.arraycopy(heights, 0, temp, 0, len);
        Arrays.sort(temp);
        int count = 0;
        for(int i = 0; i < len; i++){
            if(temp[i]!=heights[i])
                count++;
        }
        
        return count;
    }
}

File: code/lt1051.cpp
#include <vector>
#include <algorithm>

using namespace std;

class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> nHeights = heights;
        sort(nHeights.begin(), nHeights.end());
        int num = 0;
        for(int i = 0; i < heights.size(); i++)
            if(heights[i]!=nHeights[i])
                num++;
        return num;
    }
};